### PR TITLE
Use macro-compat

### PR DESCRIPTION
**Pull request by aoiroaoino**
ref. #248

@milessabin @philwills @julien-truffaut many thanks! :smile:

### CODE REVIEW

File: macro/src/main/scala/monocle/macros/GenIso.scala [hunk 1]
```diff
@@ -59,21 +60,22 @@ private object GenIsoImpl extends MacrosCompatibility {
...
 
-  def genIso_unit_impl[S: c.WeakTypeTag](c: Context): c.Expr[Iso[S, Unit]] = {
+  def genIso_unit_impl[S: c.WeakTypeTag]: c.Expr[Iso[S, Unit]] = {
     import c.universe._
 
     val sTpe = weakTypeOf[S]
 
     if (sTpe.typeSymbol.isModuleClass) {
       val table = c.universe.asInstanceOf[SymbolTable]
-      val obj = makeAttributedQualifier(c)(table.gen, sTpe)
```
Comment (milessabin) [inline macro/src/main/scala/monocle/macros/GenIso.scala]: Would be worth adding `Context#internal.gen.mkAttributedQualifier` to macro-compat too.

**Comment by julien-truffaut** [comment]
Great!

File: macro/src/main/scala/monocle/macros/GenIso.scala (hunk 1 above)
Comment (aoiroaoino) [inline macro/src/main/scala/monocle/macros/GenIso.scala]: ok, I will send a pull request :+1:

**Comment by julien-truffaut** [comment]
:+1:

**Pull request merged**